tests/src/Functional/CacheTest.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_contact_forms\Functional;
+
+use Drupal\contact\Entity\ContactForm;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\user\Entity\Role;
+use Drupal\user\RoleInterface;
+
+/**
+ * Tests the corporate contact form rendering under cache.
+ */
+class CacheTest extends BrowserTestBase {
```

Comment:
Why are we making a brand new test? We have other functional tests that where we can ensure this cache gets invalidated...

Reply:
moved into MessageFormTest.php